R/module_teal.R

```diff
@@ -113,6 +113,12 @@ ui_teal <- function(id,
       tags$div(
         footer,
         teal.widgets::verbatim_popup_ui(ns("sessionInfo"), "Session Info", type = "link"),
+        br(),
+        if (!inherits(session, "MockShinySession")) {
```

Comment:
similar to https://github.com/insightsengineering/teal/pull/1232/files#r1611154373

Please consider a new util function

Reply:
it is a one liner that exists in 2 places. I will leave it as it is.

Reply:
ok

Reply:
at the end I created 3 helper functions
https://github.com/insightsengineering/teal/pull/1232/files#diff-60bef5af43a1bd541299ce9bceeb226afb23deaa5e637b145c85e05ffd552b0cR453-R463